_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,240 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are instruments to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [HTML element][] if:
+
+- **(changed)** the `innerText` property of the [element][html element] changes; and
+- **(no child changed)** the [element][html element] does not have [children][child] whose `innerText` property also changes; and
+- **(not alone)** the [element][html element] has [sibling][] or [ancestor][] elements with non-empty `innerText` properties whose value is different from the `innerText` of the test target; and
```

Comment:
I like this a lot. Took me a few minutes to realise what this does though. Maybe add a note to explain this? I'm not sure this sibling thing is necessary though. If it has a sibling with a different innerText, it's parent will also have a different innerText.